Placeeo.xcodeproj/xcuserdata/synecdochenoire.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Bucket
+   uuid = "173CCDB8-F03E-44FC-9314-354A2F8912B8"
+   type = "1"
+   version = "2.0">
+</Bucket>
```

Comment:
breakpoints mustn't be commited. please remove this change and update the .gitignore to avoid this situations in future. 
you can just copy-paste this autogenerated one: https://www.toptal.com/developers/gitignore/api/xcode,swift
please create a ticket for updating the .gitignore file

Reply:
done!